app/parse.py

```diff
@@ -8,8 +14,39 @@ class Quote:
     tags: list[str]
 
 
+def get_quote(quote: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote.select_one(".text").text,
+        author=quote.select_one(".author").text,
+        tags=[text.text for text in quote.select(".tag")]
+    )
+
+
+def get_all_quotes_from_page(num_of_page: int) -> [Quote]:
+    page = requests.get(f"{BASE_URL}/page/{num_of_page}/").content
```

Comment:
wrap it in try except. If there won't be internet connection or no content, your code will fail